openhantek/src/glscope.cpp

```diff
@@ -129,7 +132,7 @@ void GlScope::initializeGL() {
 
     auto program = std::unique_ptr<QOpenGLShaderProgram>(new QOpenGLShaderProgram(context()));
 
-    const char *vshaderES = R"(
+    const char *vShader_1_0 = R"(
```

Comment:
Don't rename those please. You missunderstood the OpenGL shader version schema. OpenGL is something different to OpenGL ES and those two standards have different shader versions. OpenGL ES shader version 1.0 originates from OpenGL shader version 1.2 but was slightly modified. It's complicated. The names therefore should be _ES and _Desktop.

Reply:
Thanks for explaining, my previous OpenGL experience is limited to embedded systems. Will revert.